SpatialGDK/Documentation/known-issues.md

```diff
@@ -26,3 +26,4 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Level streaming might affect performance in certain scenarios. | 2019-03-26 | [UNR-1163](https://improbableio.atlassian.net/browse/UNR-1163) | Select the **Query Based Interest Enabled** checkbox in the SpatialOS GDK for Unreal Runtime Settings to enable or disable level streaming. If the **Query Based Interest Enabled** checkbox is ticked, level streaming will be supported, but this might affect performance in certain scenarios and is currently being investigated. |
 | Changing some SpatialOS settings after launching a local deployment will invalidate the currently launched deployment (e.g. Changing the number of servers, or toggling using Query Based Interest), as these settings modify the launch configuration. | 2019-03-26 | [UNR-1190](https://improbableio.atlassian.net/browse/UNR-1190) | Manually stop and start your deployment after you make any changes that alter the launch configuration. |
 | The GDK ignores the `bHidden` specifier for actors | 2019-04-16 | [UNR-1182](https://improbableio.atlassian.net/browse/UNR-1182) |  Do not rely on `bHidden`, set `bReplicates` to the correct value instead. |  
+| Schema generation removes non-alphanumeric characters during generation, that might lead to naming collisions. | 2019-04-23 | [UNR-1322](https://improbableio.atlassian.net/browse/UNR-1322) |  Use alphanumeric characters only. |  
```

Comment:
```suggestion
| Schema generation removes non-alphanumeric characters. This can lead to name collisions in subobjects, properties, and RPCs, which is not supported and will result in an error. | 2019-04-23 | [UNR-1322](https://improbableio.atlassian.net/browse/UNR-1322) |  Ensure that names are not identical with non-alphanumeric removed. For example, `My Property` and `MyProperty` will result in an error, but `My Property` and `MyProperty2` will not. |  
```